tests/testWPAuth0Options.php

```diff
@@ -48,6 +48,22 @@ public function testThatDefaultsWork() {
 		}
 	}
 
+	/**
+	 * Test that a missing option (considered an empty value) is not updated to a default "on" value.
+	 */
+	public function testThatGetOptionsDoesNotUseDefaultsWhenNotEmpty() {
+
+		$db_options = get_option( self::OPTIONS_NAME );
+		unset( $db_options['auto_login'] );
```

Comment:
any reason why you chose these 2 options? would it make sense to first assert they have a value, then unset it. Then call update. Then assert still empty?

Reply:
These options have a default of `1` but, when saved, they were being removed from the options array instead of settings to a false-y value like `0`. This isn't optimal but works fine as the `get()` would send back something false-y. But if the defaults change (as they did in this version) and an update is run, then the missing options would be updated to defaults, which could be different.